### PR TITLE
feat(changesets): move avatar from alpha package to automatic release [AHOY-3166]

### PR DESCRIPTION
**Pull request by Annmary12**
# Purpose of PR
This PR moves Avatar from the alpha package to the automatic release.

## PR Checklist

- [x] I have read the relevant `readme.md` file(s)
- [x] All commits follow our [Git commit message convention](https://github.com/contentful/forma-36/tree/main/packages/forma-36-react-components#commits)
- [x] Tests are added/updated/not required
- [x] Tests are passing
- [x] Storybook stories are added/updated/not required
- [x] Usage notes are added/updated/not required
- [x] Has been tested based on [Contentful's browser support](https://www.contentful.com/faq/about-contentful/#which-browsers-does-contentful-support)
- [x] Doesn't contain any sensitive information

### CODE REVIEW

**Comment by denkristoffer** [comment]
@Annmary12 Can you rebase on top of `main`?

@massao Does this look right?

**Review by massao — COMMENTED** [review]
You also need to:
- Add the avatar to the `f36-components` package, as a dependency and as an export, also updating the example on how to import.
- Also remove the alpha avatar from the `SandpackRenderer.tsx`, [here](https://github.com/contentful/forma-36/blob/8c924ef8a4d81e380ddb58c71d02f2d9cfef9b87/packages/website/components/Playground/SandpackRenderer.tsx#L98)
- Remove from LiveEditor [here](https://github.com/contentful/forma-36/blob/8c924ef8a4d81e380ddb58c71d02f2d9cfef9b87/packages/website/components/LiveEditor/ComponentSource.tsx#L44)
- Add a changeset so it automatically updates the version to match the other packages and bump the versions.

I think that's it, but will take another look later again.

**Comment by Annmary12** [comment]
> You also need to:
> 
> * Add the avatar to the `f36-components` package, as a dependency and as an export, also updating the example on how to import.
> * Also remove the alpha avatar from the `SandpackRenderer.tsx`, [here](https://github.com/contentful/forma-36/blob/8c924ef8a4d81e380ddb58c71d02f2d9cfef9b87/packages/website/components/Playground/SandpackRenderer.tsx#L98)
> * Remove from LiveEditor [here](https://github.com/contentful/forma-36/blob/8c924ef8a4d81e380ddb58c71d02f2d9cfef9b87/packages/website/components/LiveEditor/ComponentSource.tsx#L44)
> * Add a changeset so it automatically updates the version to match the other packages and bump the versions.
> 
> I think that's it, but will take another look later again.
> 
> @Annmary12 can you just add a changeset, the last point, so the package get correctly release, we can do the other points in a follow up once we have a stable version of avatar release.

Done, please review

File: .changeset/config.json [hunk 1]
```diff
@@ -7,7 +7,6 @@
     }
   ],
   "ignore": [
-    "@contentful/f36-avatar",
     "@contentful/f36-image",
```
Comment (massao) [inline .changeset/config.json:10]: We need to remove the image and the navbar from the ignore, to make the changeset config valid

Comment (Annmary12) [reply .changeset/config.json]: done

**Review by massao — APPROVED** [review]
LGTM

**Pull request merged**